kalman_watch/monitor_win_perigee.py

```diff
@@ -483,15 +483,21 @@ def get_kalman_drops_per_minute(mon: MonDataSet) -> tuple[np.ndarray, np.ndarray
     dt_mins : np.ndarray
         Array of dt_min values (minutes since perigee) for each minute
     kalman_drops : np.ndarray
-        Array of number of kalman drops per minute
+        Array of number of kalman drops per minute scaled
     """
     bins = np.arange(-30, 31, 1.025)
```

Comment:
If these bins are basically perigee-relative times, and we're talking about going out to +32 now anyway, we might want to increase the range.